Nikita_Konkov_Application/CSharpBasicsForLessonFour/CSharpBasicsForLessonFour/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Microsoft.VisualBasic;
+using System;
+using System.Linq;
+using System.Runtime.Serialization.Formatters;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+
+new Homework_2().Task_3();
+
+class Homework_2
+{
+    public void Task_1()
+    {
+        //int enteredNumber = Convert.ToInt32(Console.ReadLine());
+
+        //if (enteredNumber % 2 == 0 && enteredNumber % 5 == 0)
+        //{
+        //    Console.WriteLine("TuttiFrutti");
+        //}
+        //else if (enteredNumber % 5 == 0)
+        //{
+        //    Console.WriteLine("Frutti");
+        //}
+        //else if (enteredNumber % 2 == 0)
+        //{
+        //    Console.WriteLine("Tutti");
+        //}
+        //else
+        //{
+        //    Console.WriteLine("Сan’t be divided on 2 or 5");
+        //}
+
+        //Console.ReadLine();
+
+    }
+    public void Task_2()
+    {
+
+        //int lessnum = Convert.ToInt32(Console.ReadLine());
+        //int biggernum = Convert.ToInt32(Console.ReadLine());
+
+        //for (int i = lessnum; i <= biggernum; i++)
+        //{
+        //    if (i % 2 == 0 && i % 5 == 0)
+        //    {
+        //        Console.WriteLine("TuttiFrutti");
+        //    }
+        //    else if (i % 5 == 0)
+        //    {
+        //        Console.WriteLine("Frutti");
+        //    }
+        //    else if (i % 2 == 0)
+        //    {
+        //        Console.WriteLine("Tutti");
+        //    }
+        //    else
+        //    {
+        //        Console.WriteLine("Сan’t be divided on 2 or 5");
+        //    }
+        //}
+
+        //Console.ReadLine();
+
+    }
+    public void Task_3()
+    {
+        int firstNum = Convert.ToInt32(Console.ReadLine());
+        int secondNum = Convert.ToInt32(Console.ReadLine());
+
+        if (firstNum > secondNum)
```

Comment:
just to decrease number of code duplication in this method you can initialize 2 more variables. 
var biggerNum = 0;
var smallerNum = 0;
(if firstNum >= secondNum)
{
biggerNum = firstNum;
smallerNum = secondNum;
}
else 
{
biggerNum = secondNum;
smallerNum = firstNum;
}